ansible_collections/arista/avd/requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-ansible>=2.10,<2.11
+ansible-core>=2.11.3
```

Comment:
@ClausHolbechArista - I believe we are required by Automation hub to set an upper limit
So I recommend updating to this: `ansible-core>=2.11.3,<2.12`

Reply:
It is not a requirement to have a max set. We didn't until recently. We added it in #732 because of issues with Ansible base vs core installations.